components/converter/TransactionBadge.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import Anchor from '../Anchor/Anchor'
-import environment from 'lib/environment'
 import { shortenAddress } from 'lib/web3-utils'
+import { getEtherscanHref } from 'lib/utils'
 
 function TransactionBadge({ transactionHash }) {
   return (
```

Comment:
@bpierre I'm wondering with this Badge, since it's always a link, should we have the underline and text-colour changes on hover? I think in other apps we only change the pointer when hovering over a Badge.

Reply:
Yes we usually don’t use any hover state for links: https://ui.aragon.org/link/

For the underline, the general idea is to use them for external links, and not for internal links or when used as a text button.